packages/manager/modules/vrack/src/dashboard/vrack.less

```diff
@@ -104,4 +104,8 @@
   &__empty-description-text {
     margin: 2.1875rem 0 0.9375rem 0;
   }
+
+  .vrack__service-fetching {
+    float: right;
+  }
```

Comment:
**nitpick:** Use `float-right` bootstrap class instead.

Reply:
added float-right instead of this specific css